app/models/video.rb

```diff
@@ -97,6 +97,10 @@ def fill_missing_fields
     write_attribute(:amara_id, YoutubeReader::amara_id(self.youtube_id))
   end
 
+  def similar_videos
```

Comment:
Can probably just name this method `similar`, since video is implied from the model.